rasa/nlu/model.py

```diff
@@ -385,7 +385,10 @@ def parse(
             output["text"] = ""
             return output
 
-        message = Message(text, self.default_output_attributes(), time=time)
+        data = self.default_output_attributes()
+        data[TEXT] = text
```

Comment:
optional: does it make sense to make `text` an optional parameter of `default_output_attributes`? Then you could write:
```python
data = self.default_output_attributes(text=text)
```

Reply:
It is used differently in other places and I'd rather leave it in than having to fix half the files (which seems to be my hobby these days 😅)